package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "battleship-game",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ngodi/battleship-game.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ngodi/battleship-game/issues"
+  },
+  "homepage": "https://github.com/ngodi/battleship-game#readme",
+  "devDependencies": {
+    "jest": "^24.9.0",
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.3.10"
+  },
+  "dependencies": {
```

Comment:
**[ REQUIRED ]** Also please remember to add all the necesary scrips to build your your project, as required [HERE](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622118/): 
![image](https://user-images.githubusercontent.com/46115725/70768496-043b0f00-1d2b-11ea-8697-258d8b49ca68.png)

It could be something like this:
![image](https://user-images.githubusercontent.com/46115725/70768540-3b112500-1d2b-11ea-8016-39290fe72237.png)